frontend/src/components/ResultsList.js

```diff
@@ -42,15 +42,19 @@ const sampleJSON = {
         },
 }
 
-const ResultsList = () => (
+const ResultsList = ({difficulty}) => (
     <Fragment>
         <div className="results-container">
-            {Object.keys(sampleJSON).map(app => 
-                <Result 
-                    name={sampleJSON[app].name} 
-                    description={sampleJSON[app].description} 
-                    link={sampleJSON[app].link} 
-                />
+            {Object.keys(appIdeas).map(function (app) { 
+
+                if (appIdeas[app].difficulty === difficulty)
```

Comment:
For the sake of readability, this line is confusing. Please, consider renaming either the state variable from `difficulty` or the `difficulty` property of objects to something to avoid the naming confusion.